tests/data/wflow_v0x/sbm/wflow_sbm_v1.toml

```
@@ -127,6 +127,11 @@ name = "Q"
 map = "gauges"
 parameter = "river_water__instantaneous_volume_flow_rate"
 
+[[output.netcdf_scalar.variable]]
+name = "lake-volume"
+map = "reservoir_location__count"
+parameter = "lake_water__instantaneous_volume"
```

I see I forgot to change this in the lake/reservoir PR.... Sorry about that. It should be:

```suggestion
parameter = "reservoir_water__instantaneous_volume"
```

To change this you have to update the `additional_variables` conversion dict here: https://github.com/Deltares/hydromt_wflow/blob/c6065ec384164840d8b9239cdce1c2ef65ffa8e8/hydromt_wflow/version_upgrade.py#L317

Change:
```
"lateral.river.reservoir.volume": "reservoir_water__instantaneous_volume",
"lateral.river.reservoir.totaloutflow": "reservoir_water~outgoing__volume_flow_rate",  # noqa : E501
"lateral.river.lake.storage": "lake_water__instantaneous_volume",
"lateral.river.lake.totaloutflow": "lake_water~outgoing__volume_flow_rate",
```

to:
```
"lateral.river.reservoir.volume": "reservoir_water__instantaneous_volume",
"lateral.river.reservoir.totaloutflow": "reservoir_water~outgoing__volume_flow_rate",  # noqa : E501
"lateral.river.lake.storage": "reservoir_water__instantaneous_volume",
"lateral.river.lake.totaloutflow": "reservoir_water~outgoing__volume_flow_rate",
```

---

I've made the change, but I'm wondering what would happen if someone has both in their system. Would that give issues, because there will be 2 the same keys in their configuration after upgrading?

---

The principal is that the name is not the same for netcdf_variable. So the model will run and users may save the same info twice. So not really an issue, I think. We do not support entire conversion too as we did not list all possible outputs of wflow as that would be too much. We decided to support only the most common ones.